test/test_bq_billing_ar_batch.py

```diff
@@ -168,7 +168,7 @@ async def test_get_ar_guid_by_batch_id_no_data(self):
         self.bq_result.result.return_value = []
 
         # test get_ar_guid_by_batch_id function
-        ar_guid = await self.table_obj.get_ar_guid_by_batch_id(batch_id)
+        _st, _ed, ar_guid = await self.table_obj.get_ar_guid_by_batch_id(batch_id)
```

Comment:
```suggestion
        _, _, ar_guid = await self.table_obj.get_ar_guid_by_batch_id(batch_id)
```

Reply:
Or this, but it might confuse people haha
```suggestion
        *_, ar_guid = await self.table_obj.get_ar_guid_by_batch_id(batch_id)
```

Reply:
Good point, changed to _,_, to not confused people :-)